npm/vite-dev-server/package.json

```diff
@@ -17,8 +16,7 @@
   },
   "dependencies": {
     "debug": "4.3.3",
-    "find-up": "6.3.0",
-    "local-pkg": "0.4.1",
+    "find-up": "5.0.0",
```

Comment:
One thing to consider here is that these packages are always going to be consumed through our loader process since they are imported in the config (which is processed in the child process we spin up). Not quite sure of all the implications of this, but if we always transpile the config to commonjs then it might factor into how we decide to bundle these packages.

I'm seeing the need for a clearly defined tech-brief just talking through all of this!

Reply:
Ah yes, we'd need to change our development workflow to transpire this on the fly if we wanted to write pure ESM (probably not a big deal, but more work than really makes sense now).

That said, unless we eventually do commit to it, we will likely be stuck on ESM forever (like most of the ecosystem). We finally got the module system we all dreamed off, but it came 10 years too late :(